packages/cactus-plugin-ledger-connector-iroha/src/main/json/openapi.json

```diff
@@ -231,6 +236,82 @@
       "PrometheusExporterMetricsResponse": {
         "type": "string",
         "nullable": false
+      },
+      "IrohaSocketSessionEvent": {
```

Comment:
@outSH `IrohaSocketSessionEvent` is a thing not an action that an endpoint would undertake (e.g. it's a noun not a verb). Naming it like this crowds out everyone else from adding any other future endpoints that are SocketIO based because then we would have endpoints called `IrohaSocketSessionEvent` and `DoStuffV1` and `DoOtherStuffV1` but they would all be based on socket sessions and their events so people could end up getting pretty confused over their relationship to each other. I'd recommend naming it `WatchBlocksV1` instead, then it describes what it does instead of only hinting at the transport it uses and saying nothing about the actual operation that it undertakes.

Reply:
@petermetz Yeah I've changed them to match the other connectors.

Reply:
@outSH Thank you very much!